src/tests/account/test_eth_account.cairo

```diff
@@ -583,6 +610,32 @@ fn test__set_public_key() {
 // Helpers
 //
 
+fn get_accept_ownership_signature() -> Span<felt252> {
+    let mut output = array![];
+
+    // 0x02357636c7266194923ea0b270bd563853c1fdea875b3cc6567f6d08cbccc3be =
+    // PoseidonTrait::new()
+    //             .update_with('StarkNet Message')
+    //             .update_with('accept_ownership')
+    //             .update_with(ACCOUNT_ADDRESS())
```

Comment:
Isn't it weird that `ACCOUNT_ADDRESS` is `0`? I know it's just for tests and AFAICT it doesn't sully the integrity of the tests we have, but IMO this should be non-zero. Do you agree?

Reply:
Good catch, and I agree.

Reply:
Fixed!